apps/tup-ui/src/pages/Login/Layout.global.css

```diff
@@ -0,0 +1,4 @@
+.o-site {
+  display: grid;
+  height: 100vh;
+}
```

Comment:
❓ I am not sure what classname to use… I borrowed `.o-site` from Core CMS and Design Safe (which adapted, and improved, it from Core CMS). But it's not complex enough to merit a component.